src/aks-preview/azext_aks_preview/tests/latest/recordings/test_aks_addon_disable_confcom_addon.yaml

```diff
@@ -60,7 +60,7 @@ interactions:
       User-Agent:
       - AZURECLI/2.53.0 azsdk-python-azure-mgmt-resource/23.1.0b2 Python/3.8.10 (Linux-5.15.0-1051-azure-x86_64-with-glibc2.29)
     method: GET
-    uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/clitest000001?api-version=2022-09-01
+    uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/clitest000001?api-version=2024-11-01
```

Comment:
@FumingZhang - Do you understand why these changes were required?

Do you have an outstanding PR to tackle these that Sophie could rebase onto?

Reply:
yep, I've applied fix for failed test cases in PR https://github.com/Azure/azure-cli-extensions/pull/8798

Reply:
please rebase from main to drop unrelated changes